src/osd/OSD.cc

```diff
@@ -2268,12 +2268,15 @@ int OSD::set_numa_affinity()
   string back_iface = pick_iface(
     cct,
     cluster_messenger->get_myaddrs().front().get_sockaddr_storage());
-  int r = get_iface_numa_node(front_iface, &front_node);
-  if (r >= 0) {
+  int r = get_bond_numa_node(front_iface, &front_node);
+  if (r < 0) {
+    r = get_iface_numa_node(front_iface, &front_node);
+  }
+  if (r >= 0 && front_node >= 0) {
     dout(1) << __func__ << " public network " << front_iface << " numa node "
 	      << front_node << dendl;
     r = get_iface_numa_node(back_iface, &back_node);
```

Comment:
I don't understand why we check for bonding on the front but not the back... if we make the existing get_iface_numa_node understand bonded interfaces, will that still do the right thing?